components/TableElements/ClassTableStyle.ts

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components'
+
+export const TableTitle = styled.h2`
+  text-align: center;
+`
+
+export const TableContainer = styled.table`
+  margin: 0 auto;
+  min-height: 100%;
+  width: 95%;
+`
+
+export const TableHeader = styled.thead`
+  height: 40px;
+  background-color: ${({ theme }) => theme.bg.secondary};
+  color: ${({ theme }) => theme.text.primary};
+  text-transform: capitalize;
+`
+export const TableRow = styled.tr`
+  height: 80%;
+  text-align: center;
+`
+export const TableField = styled.td`
+  height: 50px;
+  width: 200px;
+  background-color: ${({ theme }) => theme.bg.quarternary};
+  vertical-align: center;
+  text-transform: capitalize;
+`
+
+export const SmallButton = styled.button`
+  border-radius: 5px;
+  background: ${({ theme }) => theme.bg.secondary};
+  padding: 1rem;
+  color: ${({ theme }) => theme.text.primary};
+  width: 70px;
+  font-size: 1.25rem;
+  outline: none;
```

Comment:
Same comment as on the other `SmallButton` component that you've created multiple times 😆